dev/functional/tests/cypress/page-objects/Institution-objects/ManageUsersObject.ts

```diff
@@ -44,7 +44,7 @@ export default class ManageUsersObject {
   }
 
   searchButton() {
-    return cy.get(".v-btn--tile");
+    return cy.get('[tile=""]');
```

Comment:
It seems to be way too generic.

Reply:
Changes have been made to the selector.
<img width="299" alt="image" src="https://user-images.githubusercontent.com/98920571/170398785-543cf81f-2010-4226-b4fb-041304a578dd.png">